src/ArborX_DBSCAN.hpp

```diff
@@ -297,8 +297,12 @@ dbscan(ExecutionSpace const &exec_space, Primitives const &primitives,
     {
       // Perform the queries and build clusters through callback
       using CorePoints = Details::CCSCorePoints;
+      // FIXME_NVCC(11.0-11.3): this HalfTraversal typedef is a workaround
+      using HalfTraversal = Details::HalfTraversal<
+          decltype(bvh), Details::FDBSCANCallback<UnionFind, CorePoints>,
+          Details::WithinRadiusGetter>;
```

Comment:
(Not tested)
```suggestion
      // Workaround NVCC 11.0-11.3 internal compiler error
      #if defined(KOKKOS_COMPILER_NVCC) && (KOKKOS_COMPILER_NVCC < 1140)
      using HalfTraversal = Details::HalfTraversal<
          decltype(bvh), Details::FDBSCANCallback<UnionFind, CorePoints>,
          Details::WithinRadiusGetter>;
      #else
      using HalfTraversal = Details::HalfTraversal;
      #endif
```

Reply:
I don't think that the
```
using Details::HalfTRaversal
```
alias valid since it's a template (and you can't put the template declaration here at block scope).